bigc/resources/orders.py

```diff
@@ -110,6 +110,10 @@ def create_shipment(self, order_id: int, *, order_address_id: int, items: list[d
 
         return self._api.v2.post(f'/orders/{order_id}/shipments', json=payload)
 
+    def update_shipment(self, order_id: int, shipment_id: int, **kwargs: dict) -> dict:
```

Comment:
I don't think we typehint kwargs typically.
```suggestion
    def update_shipment(self, order_id: int, shipment_id: int, **kwargs) -> dict:
```

Reply:
Agreed, plus [this typehint is incorrect anyways](https://peps.python.org/pep-0484/#arbitrary-argument-lists-and-default-argument-values). Kwargs/args typehints should be for the value of each item in the args/kwargs. For example:
```
def my_func(**kwargs: int):
   ...
```
defines a function which can do something like `my_func(a=1, b=2, c=3, aadfjhdfakjdfkahdfj=120)`

Reply:
I figured it was probably incorrect but wasn't 100% sure myself, thanks for that link.